### ISSUE TITLE
Split panes based on size of current pane

### ISSUE THREAD

**Issue by Haltarys** [gnome-terminator/terminator#772]
**Is your feature request related to a problem? Please describe.**
I like the idea of splitting terminal windows into panes but I don't want to manually have to split horizontally or vertically. I'd like terminator to decide that for me.

**Describe the solution you'd like**
I'd like a new keyboard shortcut `Split based on size` that will automatically determine whether to split horizontally or vertically based on the size of the current pane. If the current pane's height is greater than its width, split horizontally. If the current pane is wider than it is high, split vertically.  
We should ideally be able to mesure the height and width in pixels or in number of characters long or number of lines high.

**Describe alternatives you've considered**
I think `tmux` might be able to do that but I like the simplicity of Terminator better.

**Additional context**
Basically, this feature is from Windows Terminal. And it's great.

**Comment by mattrose** [comment]
There is a new "Split Auto" menu entry currently in master that will be released in the next release.  If you want to try it out now you can check out master.

**Comment by Haltarys** [comment]
Great! Will this be released on the Ubuntu Store as well?
